workspaces/liferay-sample-workspace/client-extensions/liferay-sample-fds-cell-renderer/package.json

```diff
@@ -1,16 +1,26 @@
 {
 	"dependencies": {
-		"@liferay/dxp-7.4": "*",
-		"@liferay/js-api": "0.2.0-pre.0"
+		"@liferay/js-api": "^0.2.0-pre.0",
+		"react": "16.12.0",
+		"react-dom": "16.12.0"
 	},
 	"description": "Liferay Sample Frontend Data Set Cell Renderer",
+	"devDependencies": {
+		"@types/react": "^18.0.33",
+		"@types/react-dom": "^18.0.11",
+		"babel-loader": "^9.1.2",
+		"ts-loader": "^9.4.2",
+		"typescript": "^5.0.3",
+		"webpack": "^5.78.0",
+		"webpack-cli": "^5.0.1",
+		"webpack-merge": "^5.8.0"
+	},
 	"main": "./src/index.tsx",
 	"name": "liferay-sample-fds-cell-renderer",
 	"scripts": {
-		"build": "liferay build",
-		"clean": "liferay clean",
-		"deploy": "liferay deploy",
-		"start": "liferay start"
+		"build": "webpack --config webpack.prod.js",
```

Comment:
Can we avoid using webpack (and react) in this example as we said? Just typescript building with ES2020 target would be enough I think...

Because, if we are showcasing the FDS cell renderer feature we don't need react for anything. The only thing it adds is noise.

So, until someone asks for it, we only provide pure JS samples.

Reply:
I'm with this idea if we are able to keep sample working

Reply:
It doesn't need webpack for anything. I guess we will have to end up adding an example with webpack because people will ask for it, but I see value in providing the leanest example possible for each feature. It will allow technical people to understand the nature of each CX.